client/src/layouts/__tests__/__snapshots__/index-test.js.snap

```diff
@@ -148,7 +151,7 @@ exports[`renders Component default correctly 1`] = `
     </div>
   </div>
   <main
-    className="component"
+    className="component "
```

Comment:
it's because of ternary condition.

```js
<main
      className={`${classes.component} ${
        isCollapsed ? classes.isCollapsed : ""
      }`}
```

Reply:
I can use https://github.com/JedWatson/classnames to avoid this space

Reply:
Nah, the space is fine. Just wondering, thanks!